Makefile

```diff
@@ -5,8 +5,10 @@ TARGET_DIR := /usr/bin
 CONFIG_DIR := /etc
 
 define gen_manpages
-	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide-msg.1.md -o $(ROOT_DIR)/man/oxide-msg.1
 	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide.1.md -o $(ROOT_DIR)/man/oxide.1
+	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide-bar.1.md -o $(ROOT_DIR)/man/oxide-bar.1	
+	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide-config.1.md -o $(ROOT_DIR)/man/oxide-config.1
+	pandoc --standalone --to man $(ROOT_DIR)/man/src/oxide-msg.1.md -o $(ROOT_DIR)/man/oxide-msg.1
```

Comment:
You could just do this for all files in the folder ?:)
so a new file would not need to be added in here ?:)

Reply:
@FelixSchladt ?

Reply:
i tried but i dont think pandoc offers this functionality...

I could write a small shell script but in the end manual adding of files is required anyway in the install/uninstall part and I dont think there is gonna be many more manpages in the future so i see no value